components/tensorboard-controller/controllers/tensorboard_controller.go

```diff
@@ -268,6 +269,15 @@ func generateDeployment(tb *tensorboardv1alpha1.Tensorboard, log logr.Logger, r
 								},
 							},
 							VolumeMounts: volumeMounts,
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
is this enough? too low and we would be oversubscribing.

Reply:
Yes this has been enough when we run tensorboards on the Kubeflow cluster through pipeines.